packages/dx-grid-core/src/plugins/grouping-panel/helpers.ts

```diff
@@ -1,13 +1,17 @@
-export const groupingPanelItems = (columns, grouping, draftGrouping) => {
+import { GroupingPanelItemsFn } from '../../types/';
+
+export const groupingPanelItems: GroupingPanelItemsFn = (
+  columns, grouping, draftGrouping,
```

Comment:
Why we are using in some places `PureComputed` and in some places, we are using a custom type based on the `PureComputed`? I think we should choose one style. Or do not I understand something?

Reply:
When there are more than 3 arguments a type is usually introduced to improve readability. But in some places it's faster to define function type in the same line as a function